package.json

```diff
@@ -8,7 +8,7 @@
     "less": "~2.7.2",
     "less-plugin-autoprefix": "~1.4.2",
     "less-plugin-clean-css": "~1.5.0",
-    "reveal.js": "~3.5.0",
+    "reveal.js": "~3.7.0",
```

Comment:
ok, this is actually a big change... we are going straight from 3.5.0 to 3.7.0.
We should smoke test everything else is working reasonably well with this new version... for instance, the chalkboard, the speaker notes, etc...

Reply:
you're right, I won't have enough time, not enough confidence in my test setup, to test this thoroughly enough at this point, so I'll just back off to 3.5.0 again

Reply:
Thanks, pretty happy to upgrade in the future after doing some depth testing.
Let's capture that in a new issue so we do not forget about this.